docs/api/session.md

```diff
@@ -823,6 +823,68 @@ app.whenReady().then(() => {
 })
 ```
 
+#### `ses.setBluetoothPairingHandler(handler)` _Windows_ _Linux_
```

Comment:
suggestion: Would it be useful to include a bit about why this isn't needed on macOS?